mobsfscan/rules/patterns/ios/swift/swift_rules.yaml

```diff
@@ -70,9 +70,12 @@
 - id: ios_biometric_bool
   message: Biometric authentication should be based on Keychain, not based on bool.
   input_case: exact
-  pattern: \.evaluatePolicy\(.deviceOwnerAuthentication
+  pattern:
+    - \.evaluatePolicy\(.deviceOwnerAuthentication
+    - \.evaluatePolicy\(LAPolicy.deviceOwnerAuthentication
+    - evaluatePolicy:LAPolicyDeviceOwnerAuthentication
```

Comment:
This one (`LAPolicyDeviceOwnerAuthentication`)  looks like Objective C code?

https://developer.apple.com/documentation/localauthentication/lapolicy/lapolicydeviceownerauthentication?language=objc

Should this be moved to ` mobsfscan/rules/patterns/ios/objectivec/best_practices.yaml ` ?

Reply:
Yes, will take care of it.

Reply:
Have added it to `objectivec/objective_c_rules.yaml` to be consistent as the swift one was not under best practices either.

Reply:
Yes, you are right.